setup/gui/Setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+###############################
+# Declaration Area
+###############################
+
+CHOICE=""
+DE_NAME=""
+
+##############################
+# Upgrade the Machine
+##############################
+
+sudo apt-get update
+sudo apt-get upgrade
+
+##############################
+# Install Desktop Environment
+##############################
+clear
+echo -e "\n Select the Desktop Environment\n 1. Ubuntu \n 2. Xfce4"
+read -p "Enter Your Choice: " CHOICE
+
+case $CHOICE in
+  1)
+    DE_NAME="Ubuntu"
```

Comment:
Also Ubuntu is the distribution, not the DE :P

Reply:
Wew